tests/test_torchtune.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+
+import subprocess
+
+class TestAlbumentations(unittest.TestCase):
```

Comment:
rename test class to `TestTorchtune`.

Reply:
Wow, this is why we get PRs reviewed 🤦🏾‍♂️